php/class-fm-bylines.php

```diff
@@ -558,6 +562,31 @@ public function get_byline_posts( $byline_id, $type = 'author', $params = array(
 		}
 
 		/**
+		 * Get WP Query for the posts for a single byline.
+		 * Use to create loops on single byline pages.
+		 * @param int. $byline_id
+		 * @param string. $byline_type
+		 * @param array. $params
+		 * @return object. WP_Query object.
+		 */
+		public function get_byline_posts_query( $byline_id, $type, $params = array() ) {
+			$byline_id = ( empty( $byline_id ) ) ? get_the_ID() : $byline_id;
+			$type = ( empty( $type ) ) ? $this->get_byline_type() : $type;
+
+			if ( ! empty( $byline_id ) && ! empty( $type ) ) {
+				$defaults = array(
+					'post_type' => array( 'post' ),
+					'post_status' => 'publish',
+					'meta_key' => 'fm_bylines_' . sanitize_title_with_dashes( $type ) . '_' . $byline_id,
+					'suppress_filters' => false,
+				);
+				$args = wp_parse_args( $params, $defaults );
+				return new WP_Query( $args );
+			}
+			return false;
```

Comment:
I wouldn't ever return `false`; instead, `return new WP_Query;`. That way you're guaranteeing a `WP_Query` response, and `WP_Query` provides all the necessary methods with which to find out if it has posts or not. I'll also note that instantiating an empty `WP_Query` object won't add significant overhead -- it won't query the db or anything. p.s. assuming you do this, you need to update `functions.php:51`